code/datums/api.dm

```diff
@@ -633,10 +633,50 @@ proc/api_update_command_database()
 		data = null
 		return 1
 
+//Get Server Status
+/datum/topic_command/get_serverstatus_public
+	name = "get_serverstatus_public"
+	description = "Gets the serverstatus"
+	no_auth = 1
+
+/datum/topic_command/get_serverstatus/run_command(queryparams)
+	var/list/s[] = list()
+	s["version"] = game_version
+	s["mode"] = master_mode
+	s["respawn"] = config.abandon_allowed
+	s["enter"] = config.enter_allowed
+	s["vote"] = config.allow_vote_mode
+	s["ai"] = config.allow_ai
+	s["host"] = host ? host : null
+	s["players"] = 0
+	s["admins"] = 0
+	s["stationtime"] = worldtime2text()
+	s["roundduration"] = round_duration()
+	s["gameid"] = game_id
+
+	var/n = 0
+	var/admins = 0
+
+	for(var/client/C in clients)
+		if(C.holder)
+			if(C.holder.fakekey)
+				continue	//so stealthmins aren't revealed by the hub
+			admins++
+		n++
+
+	s["players"] = n
+	s["admins"] = admins
```

Comment:
Remove the admincount.
For the public its not important to know if there are admins online or not.

Reply:
grub-mkconfig -o /boot/grub/grub.cfg

Reply:
This is already exposed via. the public bot, why not have it here?

Reply:
Disagree. Players should be able to know how many staff are online.